src/dlls/mscorrc/mscorrc.rc

```diff
@@ -629,6 +629,7 @@ BEGIN
     IDS_EE_BADMARSHAL_CUSTOMMARSHALER       "Custom marshalers are only allowed on classes, strings, arrays, and boxed value types."
     IDS_EE_BADMARSHAL_GENERICS_RESTRICTION  "Generic types cannot be marshaled."
     IDS_EE_BADMARSHAL_STRING_OUT            "Cannot marshal a string by-value with the [Out] attribute."
+    IDS_EE_BADMARSHAL_BLITTABLE_GENERICS_RESTRICTION "Non-blittable generic types cannot be marshaled."
```

Comment:
Is the `IDS_EE_BADMARSHAL_GENERICS_RESTRICTION` string still relevant? Would it be more accurate to replace all instances of the old string with the new string?

Reply:
This is still being used when marshalling generic classes (modulo WinRT interfaces):
* https://github.com/dotnet/coreclr/blob/master/src/vm/mlinfo.cpp#L2022
* https://github.com/dotnet/coreclr/blob/master/src/vm/mlinfo.cpp#L2116

If there are no concerns replacing these strings (I don't see anything obvious), then I'll get a fix up.

Reply:
Fixed.